src/helper.js

```diff
@@ -47,7 +47,12 @@ const responseBody = (obj, collection) => {
 const parse_query = (req_q) => {
 	let query = {};
 	let q = {};
-	req_q.split(',').forEach(i => (q[i.split(':')[0]] = i.split(':')[1]));
+	req_q.split(',')
+	.reduce((acc, curr) => {
+		curr.includes(":") ? acc.push(curr) : acc[acc.length - 1] += ("," + curr); 
```

Comment:
This may just be my personal opinion or preference, but I would like to see an explanation for this logic, so other contributors don't need to reverse engineer it.

Reply:
Let's say the query is `name:Hello, Jsonbox,age:>15`. Normally, this would be split up into this array `["name:Hello", "Jsonbox", "age:>15" ]`. This isn't the right array and the reduce function solves this, by iterating over the false array. If the element in the array includes a `:`, the reduce function leaves it untouched, otherwise the reduce function appends the element to the previous one.
Dunno what's so hard to understand about it, much of the codebase is written in similiar way without any comments.